### PR TITLE
Add reminder UI

### PR DESCRIPTION
**Pull request by lilozz2**
Adds the Reminders after the current date from ReminderList to the dashboard UI

Part of #57, #58

Features
- [x] Reminders
- [x] Group Reminders by date
- [ ] Handle empty states

Current State:
image

### CODE REVIEW

File: src/main/java/seedu/address/ui/ReminderCard.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+     */
+
+    public final Reminder reminder;
+
+    @FXML
+    private HBox cardPane;
+    @FXML
+    private Label name;
+    @FXML
+    private Label id;
```
Comment (zhyuhan) [inline src/main/java/seedu/address/ui/ReminderCard.java]: Seems like these fields are being used. Do we need them?

File: src/main/resources/view/DashboardDisplay.fxml [hunk 2]
```diff
@@ -35,15 +36,20 @@
...
+                    <CategoryAxis fx:id="xAxis" side="BOTTOM"/>
+                </xAxis>
+                <yAxis>
+                    <NumberAxis fx:id="yAxis" side="LEFT"/>
+                </yAxis>
+            </BarChart>
+        </VBox>
+    </VBox>
+    <VBox minWidth="340" prefWidth="340" HBox.hgrow="ALWAYS" styleClass="reminder-list">
+        <Label text="Reminders" styleClass="reminder-label"/>
```
Comment (zhyuhan) [inline src/main/resources/view/DashboardDisplay.fxml]: Maybe we can use a better label, something like `Upcoming follow-ups`?

**Review by zhyuhan — APPROVED** [review]
LGTM! Left some minor comments.

**Review by KamiliArsyad — APPROVED** [review]
lgtm with nit on dashboard: the whole thing can be designed better i think

**Pull request merged**